.idea/campus-qr.iml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Have you tested checking out the whole repo from scratch? does it still work with the iml file removed?